scss/components/_button.scss

```diff
@@ -303,12 +308,31 @@ $button-transition: background-color 0.25s ease-out, color 0.25s ease-out !defau
       }
     }
 
+    // Disabled style
+    &.disabled,
+    &[disabled] {
+      @include button-disabled;
+
+      @each $name, $color in $button-palette {
+        &.#{$name} {
+          @include button-disabled($color, auto);
+        }
+      }
+    }
+
     // Hollow style
     @if $button-fill != hollow {
       &.hollow {
         @include button-hollow;
         @include button-hollow-style;
 
+        &.disabled,
+        &[disabled] {
+          &, &:hover, &:focus {
+            background-color: transparent;
+          }
+        }
```

Comment:
It might make more sense to move these styles to the mixin for hollow button. Is there a reason they're here instead? (Also, apologies for the confusion earlier!)